homeassistant/components/ezviz/camera.py

```diff
@@ -153,15 +151,6 @@ async def async_setup_entry(
         "perform_alarm_sound",
     )
 
-    platform.async_register_entity_service(
-        SERVICE_DETECTION_SENSITIVITY,
-        {
-            vol.Required(ATTR_LEVEL): cv.positive_int,
-            vol.Required(ATTR_TYPE): cv.positive_int,
-        },
-        "perform_set_alarm_detection_sensibility",
-    )
```

Comment:
This should be deprecated first, and raise an repairs issue on use.

Reply:
Have readded the service and also raises a repairs issue on use.